plugin/pkg/admission/resourcequota/admission_test.go

```diff
@@ -72,6 +72,15 @@ func validPod(name string, numContainers int, resources api.ResourceRequirements
 	return pod
 }
 
+func TestPrettyPrint(t *testing.T) {
+	input := getResourceList("100m", "5Gi")
```

Comment:
storage is not really tracked in quota, but i will add a more complex example with +2 items.